ckanext/canada/schemas/presets.yaml

```diff
@@ -2091,6 +2138,21 @@ presets:
       openness_score: 1
       value: other
 
+- preset_name: canada_resource_date_published
+  values:
+    field_name: resource_date_published
+    label:
+      en: Date Published
+      fr: Date de publication
+    help_text:
+      en: Date of formal issuance (e.g., publication) of the resource.
+      fr: Date officielle d’émission (c.-à-d. de publication) de la ressource.
+    required: true
+    # copied from date preset
+    form_snippet: date.html
+    display_snippet: date.html
+    validators: scheming_required isodate
```

Comment:
I think for resources we need to add an extra step to make sure the result is a string or this will cause a failure